tests/aws/services/ssm/test_ssm.py

```diff
@@ -153,3 +158,71 @@ def test_get_inexistent_maintenance_window(self, aws_client):
             aws_client.ssm.get_maintenance_window(WindowId=invalid_name)
         exc.match("DoesNotExistException")
         exc.match(f"Maintenance window {invalid_name} does not exist")
+
+    @markers.aws.needs_fixing
+    # remove parameters, set correct parameter prefix name, use events_create_event_bus and events_put_rule fixture,
+    # # remove clean_up, use create_sqs_events_target fixture, use snapshot
+    @pytest.mark.parametrize("strategy", ["standard", "domain", "path"])
+    def test_trigger_event_on_systems_manager_change(
```

Comment:
added a separate ticket for that

Reply:
ok, not strictly blocking but I think this one is quite important because of how we are now shipping changes to latest. Without proper optional dependencies, other services might break EventBridge forcing the service owner to jump onto a quickfix.